tf2onnx/schemas.py

```diff
@@ -77,16 +76,39 @@ def _register_all_schemas_with_history():
     return ordered_map
 
 
+def _parse_domain_opset_versions(schemas):
```

Comment:
why we don't simply check onnx package version instead of this? if the version is not for opset 9 release, then we fail it.

Reply:
then we have to keep an onnx package version -> max opset version mapping.
and every time an onnx package is released with new opset, we need to update this mapping.